openmrs/src/test/java/org/motechproject/openmrs/tasks/OpenMRSTaskDataProviderBuilderTest.java

```diff
@@ -48,12 +48,12 @@ public void generatedProviderShouldReturnNullWhenConfigsListIsEmpty() {
 
     @Test
     public void generatedProviderShouldReturnJSONWithFiveObjectsForOneConfiguration() throws IOException {
```

Comment:
please adjust method name, do not include expected number, because we will have to change method name every time someone adds new lookup to provider